modules/admin-ui-picard/app/src/components/events/partials/modals/SeriesDetails.js

```diff
@@ -0,0 +1,147 @@
+            name: 'metadata'
+        },
+        {
+            tabNameTranslation: 'EVENTS.SERIES.DETAILS.TABS.EXTENDED_METADATA',
+            name: 'extended-metadata',
+            hidden: true
+        },
+        {
+            tabNameTranslation: 'EVENTS.SERIES.DETAILS.TABS.PERMISSIONS',
+            name: 'permissions'
+        },
+        {
+            tabNameTranslation: 'EVENTS.SERIES.DETAILS.TABS.THEME',
+            name: 'theme'
+        },
+        {
+            tabNameTranslation: 'EVENTS.SERIES.DETAILS.TABS.STATISTICS',
+            name: 'statistics',
+            hidden: true
+        },
+        {
+            tabNameTranslation: 'Feeds',
+            name: 'feeds'
+        },
+    ]
+
+    const openTab = tabNr => {
+        setPage(tabNr);
+    }
+
+    return (
+        <>
+            {/* todo: role management */}
+            {/* navigation for navigating between tabs */}
+            <nav className="modal-nav" id="modal-nav">
+                <a className={cn({active: page === 0})}
+                   onClick={() => openTab(0)}>
+                    {t(tabs[0].tabNameTranslation)}
+                </a>
+                {
+                    tabs[1].hidden ?
+                        null :
+                        <a className={cn({active: page === 1})}
+                           onClick={() => openTab(1)}>
+                            {t(tabs[1].tabNameTranslation)}
+                        </a>
+
+                }
+                <a className={cn({active: page === 2})}
+                   onClick={() => openTab(2)}>
+                    {t(tabs[2].tabNameTranslation)}
+                </a>
+                <a className={cn({active: page === 3})}
+                   onClick={() => openTab(3)}>
+                    {t(tabs[3].tabNameTranslation)}
+                </a>
+                {
+                    tabs[4].hidden ?
+                        null :
+                        <a className={cn({active: page === 4})}
+                           onClick={() => openTab(4)}>
+                            {t(tabs[4].tabNameTranslation)}
+                        </a>
+
+                }
+                {feeds.length > 0 && (
+                    <a className={cn({active: page === 5})}
+                       onClick={() => openTab(5)}>
+                        {t(tabs[5].tabNameTranslation)}
+                    </a>
+                )}
+            </nav>
+
+            {/* render modal content depending on current page */}
+            <div>
+                {page === 0 && (
+                    <DetailsMetadataTab metadataFields={metadataFields}
+                                        resourceId={seriesId}
+                                        header={tabs[page].tabNameTranslation}
+                                        buttonLabel={updateSeries}
+                                        updateResource='EVENTS.SERIES.DETAILS.METADATA.REPLACE_SERIES_METADATA'/>
```

Comment:
I think `buttonLabel` and `updateResource` were switched here. shouldn't it be `buttonLabel='EVENTS.SERIES.DETAILS.METADATA.REPLACE_SERIES_METADATA'` and `updateResource={updateSeries}`?

```suggestion
                                        buttonLabel='EVENTS.SERIES.DETAILS.METADATA.REPLACE_SERIES_METADATA'
                                        updateResource={updateSeries}/>
```